packages/techdocs-cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@techdocs/cli",
   "description": "Utility CLI for managing TechDocs sites in Backstage.",
-  "version": "0.2.1",
+  "version": "0.3.0",
```

Comment:
I was looking for any changeset or changelog updates but realised we don't have that for this repo. Something we should set up maybe?

Reply:
Yes we don't have any. Deferring to https://github.com/backstage/techdocs-cli/issues/21